Add resource type dynamodb_table.

inokappa opened the pull request:
Hi, Koyama san

Otsukaresama desu.
I added DynamoDB Resource type.

### exist

```ruby
describe dynamodb('my-dynamodb-table') do
  it { should exist }
end
```

### be_active

```ruby
describe dynamodb('my-dynamodb-table') do
  it { should be_active }
end
```

### have_attribute_definition

```ruby
describe dynamodb('my-dynamodb-table') do
  it { should have_attribute_definition('my-dynamodb-table-attaribute1').attribute_type('S') }
  it { should have_attribute_definition('my-dynamodb-table-attaribute2').attribute_type('N') }
end
```

### have_key_schema

```ruby
describe dynamodb('my-dynamodb-table') do
  it { should have_key_schema('my-dynamodb-table-key_schema1').key_type('HASH') }
  it { should have_key_schema('my-dynamodb-table-key_schema2').key_type('RANGE') }
end
```

### more

```ruby
describe dynamodb('my-dynamodb-table') do
  its('provisioned_throughput.read_capacity_units') { should eq 1 }
  its('provisioned_throughput.write_capacity_units') { should eq 1 }
end
```

Please confirm. :pray:

Regards.

Yohei.

k1LoW commented:
Hi @inokappa ! Thank you for your active works!!

LGTM, but could you change `dynamodb` to `dynamodb_table` ?

k1LoW commented on lib/awspec/type/dynamodb.rb:
```diff
@@ -0,0 +1,38 @@
+module Awspec::Type
+  class Dynamodb < ResourceBase
```
Could you append `aws_resource Aws::DynamoDB::Table` ?
( So this resource type can unlock advanced use like [ec2](https://github.com/k1LoW/awspec/blob/master/doc/resource_types.md#unlock-advanced-use) )

inokappa commented:
Hi @k1LoW !

Thank you for your comment.
I changed `dynamodb` to `dynamodb_table`.
And added `aws_resource Aws::DynamoDB::Table`.

Please confirm. 🙇 

Regards.

Yohei

k1LoW commented:
@inokappa Thank you for your nice code!
LGTM!

k1LoW commented:
Released as v0.72.0.
